repository-data/webfiles/src/main/resources/site/freemarker/hee/catalog/searchresults-main.ftl

```diff
@@ -39,24 +39,24 @@
             </form>
```

Comment:
Could we comment the following active filters section & contentTypes hidden element as well ?
```
<#-- Active Filters -->
<#if selectedContentTypes?has_content>
    <div class="nhsuk-listing__active-filters nhsuk-u-margin-bottom-5">
        <#list selectedContentTypes as contentType>
            <div class="nhsuk-filter-tag nhsuk-tag" data-filter="${contentType}">
                <span>${contentTypesMap[contentType]}</span>
                <@hst.link path='/static/assets/icons/icon-close-white.svg' var="closeIcon"/>
                <img class="nhsuk-filter-tag__icon" src="${closeIcon}" alt="Remove" hidden/>
            </div>
        </#list>
    </div>
</#if>
<#-- End Active Filters -->
```

```
<#list selectedContentTypes as contentType>
    <input type="hidden" name="contentTypes" value="${contentType}">
</#list>
```

Reply:
Done